### DISCUSSION TITLE
I tried to get logs from a docker container but i see nothing

### DISCUSSION THREAD

**Discussion by billmetangmo** [logdyhq/logdy-core#33] [Q&A] [answered]
`docker logs 751aa15272c2 -f | logdy`
Any idea ?

**Comment by PeterOsinski** [comment]
Could you paste `docker logs 751aa15272c2 -f | logdy -v` output?
Is your `docker logs 751aa15272c2 -f` produces output at all?

**Reply by akash931127** [reply]
Hi @PeterOsinski , I am trying out the same, but I can't get anything
#### This is what I get in the docker logs
**NOTE: We have a mix of both json and string format logs coming in. Sorry for the redaction**
<img width="2992" height="510" alt="image" src="https://github.com/user-attachments/assets/3d789983-f225-4ac5-9c24-73c832a4cca3" />


#### This is what I see in the logdy UI
**NOTE: These are also present in the docker logs above**
<img width="1496" height="723" alt="image" src="https://github.com/user-attachments/assets/f1a6544f-bd9d-4918-bd59-78c7550682d9" />

**Reply by akash931127** [reply]
I think logdy client is not ingesting stderr logs from `docker logs`. I just checked, my application is logging all logs to stderr

**Reply by PeterOsinski** [reply]
if that's the case you can try to route both stdout and stderr to logdy
```
docker logs container 2>&1 | logdy
```

**Reply by akash931127** [reply, marked as answer]
I tried it out, last time but was not working. Seems to work now. Thanks. Works like a charm now. Need to do a -f to follow logs as well . Appreciate the help here
